src/metrics/metrics.md

```diff
@@ -1,11 +1,62 @@
 # Metrics
 
-Here we provide definitions and information about our standard metrics.
+Here we provide definitions and information about our standard metrics.  We use some terminology that is defined [here](https://docs.telemetry.mozilla.org/concepts/terminology.html).
```

Comment:
A few extra things jumped out at me reading this over again:

* Best practice is not to use "we" or "our" in technical documentation (it's not really clear "who" that means, especially in material produced by so many people)
* In general try not to use "here" when linking to things, just talk about it and linkify the concept: this is both more accessible and easier for the reader -- https://medium.com/@heyoka/dont-use-click-here-f32f445d1021 

```suggestion
This section contains definitions and information about standard metrics used at Mozilla.  You may wish to refer to the [terminology section](../concepts/terminology.html) while reading this document, in case a particular concept is not clear.
```